### ISSUE TITLE
Purchases at the hotel of the sales made but no attachments in the answer!

### ISSUE THREAD

**Issue by Hautain** [WarEmu/WarBugs#17385]
Hello,

I made two purchases at the auction house this morning: Sparkling Aurulent Ore (x2) at 350 gold pieces for one so, 700 gold pieces.

I received the messages in my auction mailbox (as you will see in the attached file).

But the Sparkling Aurulent Ore (x2) were not attached!

Thank you for correcting.

Thanking you in advance.

Yours sincerely, 
Hautain (DoK RR78, Destro clan) 

![Hautain_002](https://user-images.githubusercontent.com/68948762/88786902-02736e00-d18b-11ea-9a64-89dea3ec457c.jpg)

**Comment by Kaelang** [comment]
We don’t refund items however I’m curious, are you 100% sure these aren’t in your inventory before I start testing and digging around?

Regardless, just to confirm even if we find an issue here - no gold or items all be returned.

**Comment by Hautain** [comment]
Yes, I confirm to you that they are not in my inventory, nor were they
attached to the reply emails from the sales hotel.

Bad news...

Cordialement,
Drahcir64

**Comment by Hulkenngi** [comment]
So I just experienced something similar to this..

I just bought 13 stacks of Forgotten Soldier Insignias for 10g per stack, they all appeared as 13 mails from the auction..
But when I claimed them, only 3 stacks appeared in my backpack and all the mentioned money were taken from me.

Its not alot of money for me but I can imagine the irritation from people who tried to buy something more expensive. 

It happened on order btw.
